src/components/remote_control/include/remote_control/commands/base_command_request.h

```diff
@@ -120,6 +120,29 @@ class BaseCommandRequest
   }
 
   /**
+   * @brief IsResourceFree check resource state
+   * This is default implementation which has to be redefined for RPCs which
+   * need to manager the resources
+   * @param module_type Resource name
+   * @return True if free, otherwise - false
+   */
+  virtual bool IsResourceFree(const std::string& module_type) const {
+    UNUSED(module_type);
+    return true;
+  }
+
+  /**
+   * @brief SetResourceState changes state of resource
+   * This is default implementation which has to be redefined for RPCs which
+   * need to manager the resources
+   * @param state State to set for resource
+   */
+  virtual void SetResourceState(const Json::Value&,
+                                const ResourceState::eType state) {
+    UNUSED(state);
```

Comment:
@dev-gh probably you should also specify explicitly that first parameter is unused too or not specify that at all. Also please add description of the first param